api.planx.uk/editor/moveFlow.test.ts

```diff
@@ -0,0 +1,56 @@
+import supertest from "supertest";
+
+import { queryMock } from "../tests/graphqlQueryMock";
+import { authHeader } from "../tests/mockJWT";
+import app from "../server";
+
+beforeEach(() => {
+  queryMock.mockQuery({
+    name: "GetTeam",
+    variables: {
+      slug: "new-team",
+    },
+    data: {
+      teams: {
+        id: 1,
+      },
+    },
+  });
+
+  queryMock.mockQuery({
+    name: "UpdateFlow",
+    variables: {
+      id: 1,
+      team_id: 2,
+    },
+    data: {
+      update_flows_by_pk: {
+        id: 2,
+      },
+    },
+  });
+});
```

Comment:
This gets the test passing ✅ 
 - Return array of teams
 - Use strings for ids

```suggestion
  queryMock.mockQuery({
    name: "GetTeam",
    variables: {
      slug: "new-team",
    },
    data: {
      teams: [{
        id: "1",
      }],
    },
  });

  queryMock.mockQuery({
    name: "UpdateFlow",
    variables: {
      id: "1",
      team_id: "1",
    },
    data: {
      update_flows_by_pk: {
        id: "1",
      },
    },
  });
});
```

Reply:
thank you for spotting those finicky details in the mocks ! passing now :white_check_mark: